### ISSUE TITLE
SMS text string updates

### ISSUE THREAD

**Issue by flylikeabirdfloatlikeabuoy** [Sage-Bionetworks/covidrecoverycorps#114]
Per Vanessa's edits:

First time user who joined -  SMS should read:

"Let's get started! Enter code ______ to log in to COVID Recovery Corps."

A returned user: 
"Welcome back!  Enter code ______ to log in to COVID Recovery Corps."

**Comment by jay-hodgson** [comment]
I heard SMS login is out of phase 1 (due to sending a barcode), so is this phase 2 (or possibly never)?

**Comment by jay-hodgson** [comment]
Related to https://github.com/Sage-Bionetworks/covidrecoverycorps/issues/146

**Comment by jay-hodgson** [comment]
Initially I tried setting the 'Verify email address or phone number' SMS template to
`Let's get started! Enter code ${token} to log in to ${studyShortName}.`
And the 'Sign In' SMS template to `Welcome back! Enter code...`

But when I tried Joining using a new phone number, I was sent the `Welcome back!...` message.  Did I get these templates reversed, @alxdarksage ?

For now I just set them both to the `Let's get started! ...` template.
